VSTSAgent/VSTSAgent.psm1

```diff
@@ -303,7 +313,7 @@ function Install-VSTSAgent {
         Add-Type -AssemblyName System.IO.Compression.FileSystem -ErrorAction Stop
     }
 
-    [System.IO.Compression.ZipFile]::ExtractToDirectory($destPath, $agentFolder)
+    [System.IO.Compression.ZipFile]::ExtractToDirectory($destPath, $agentFolder, $true)
```

Comment:
Can amend this to detect if `$agentFolder` exists and if so use `$PSCmdlet.ShouldProcess` before extracting there with the force overwrite? Since the cmdlet already flags as `SupportsShouldProcess` that'll enable rich control over the operation.

Reply:
Yes, didn't think about that. I'll add that test.